graph/tags.go

```diff
@@ -241,8 +246,15 @@ func (store *TagStore) Set(repoName, tag, imageName string, force bool) error {
 	repoName = registry.NormalizeLocalName(repoName)
 	if r, exists := store.Repositories[repoName]; exists {
 		repo = r
-		if old, exists := store.Repositories[repoName][tag]; exists && !force {
-			return fmt.Errorf("Conflict: Tag %s is already set to image %s, if you want to replace it, please use -f option", tag, old)
+		if old, exists := store.Repositories[repoName][tag]; exists {
+
+			if old != img.ID && out != nil {
+				str := "The image " + repoName + ":" + tag + " already exists, renaming the old one with id " + old[:12] + " to <none>:<none>\n"
+				out.Write([]byte(str))
```

Comment:
this should be `fmt.Fprintf(out, "The image %s:%s already exists, renaming the old one with ID %s to empty string\n", repoName, tag, old[:12])`